example/gmt/Makefile.am

```diff
@@ -12,6 +12,14 @@ example_gmt_p4est_gmt_SOURCES = \
 
 LINT_CSOURCES += \
         $(example_gmt_p4est_gmt_SOURCES)
+
+bin_PROGRAMS += example/gmt/p4est_sphere_preprocessing
+
+example_gmt_p4est_sphere_preprocessing_SOURCES = example/gmt/sphere_preprocessing.c
+
+LINT_CSOURCES += \
```

Comment:
We're in the process of removing everything related to LINT. It would be ok to remove it for this example already.